src/factory.ts

```diff
@@ -0,0 +1,35 @@
+import { ConfigInterface } from './constants';
+
+export interface MessageInterface {
```

Comment:
I don't think there's any value is suffixing the interfaces with `Interface`; `Config` and `Message` are descriptive enough.

Reply:
>  I don't think there's any value is suffixing the interfaces with Interface; Config and Message are descriptive enough.

thanks for the feedback, I agree 😅was thinking same,
But I wanted to use something like IFlash, IMessage… would that be fine,
also, I have added basic docs here
https://github.com/peak-ai/flashr/blob/basic-setup/README.md
might clarify a few things.

Reply:
nvm, think without prefixing I look okay already 😃

Reply:
That's also fine. Personally I tend to lean away from the `I` prefix, but it's a practice we've adopted across Kili Ascent, so it would be nice to use it to keep things consistent.

Reply:
Hey, I read about prefixing `I` seems like the community is also moving away from it, for a couple of reasons,
Think we can stick to names without prefixing `I` convention 😅😅